RegistryManager/src/main/java/eu/neclab/ngsildbroker/registryhandler/controller/RegistryController.java

```diff
@@ -75,7 +75,7 @@ public Uni<RestResponse<Object>> getCSourceById(HttpServerRequest request,
 		return HttpUtils.validateUri(registrationId).onItem().transformToUni(t1 -> {
 			String tenant = HttpUtils.getTenantFromHeaders(headers);
 			return csourceService.getRegistrationById(registrationId, tenant).onItem().transformToUni(t -> {
-				return HttpUtils.generateReply(request, t, AppConstants.CSOURCE_URL_ID);
+				return HttpUtils.generateReply(request, t, AppConstants.CSOURCE_URL_ID,"");
```

Comment:
Please use null instead of empty string.

Reply:
done